package-lock.json

```diff
@@ -4355,6 +4355,24 @@
           "resolved": "https://registry.npmjs.org/growly/-/growly-1.3.0.tgz",
           "integrity": "sha1-8QdIy+dq+WS3yWyTxrzCivEgwIE="
         },
+        "handlebars": {
```

Comment:
was this accidentally added?

Reply:
err not sure how this got added

Reply:
can you revert the commit for the file? https://stackoverflow.com/questions/215718/how-can-i-reset-or-revert-a-file-to-a-specific-revision